.github/workflows/build.yml

```diff
@@ -0,0 +1,38 @@
+name: Fineract Gradle build
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+        TZ: Asia/Kolkata
```

Comment:
Any specific reason to select TZ as Asia/Kolkata ?

Reply:
@ptuomola  already answered this here and it makes sense. Thanks...so resolving this. see: https://github.com/apache/fineract/pull/1955#issuecomment-961172334